README.md

```diff
@@ -1,8 +1,8 @@
-# presto-admin [![Build Status](https://travis-ci.org/prestosql/presto-admin.svg?branch=master)](https://travis-ci.org/prestosql/presto-admin)
+# trino-admin [![Build Status](https://travis-ci.org/trinodb/trino-admin.svg?branch=master)](https://travis-ci.org/trinodb/trino-admin)
 
-presto-admin installs, configures, and manages Presto installations.
+trino-admin installs, configures, and manages Trino installations.
 
-Comprehensive documentation can be found at https://prestosql.github.io/presto-admin/.
+Comprehensive documentation can be found at https://github.com/trinodb/trino-admin.
```

Comment:
I don't think this is the right link for documentation. However I could not figure out where the new trino equivalent docs live for presto-admin/trino-admin. Maybe that's because the build is also not running?

Reply:
You are right, we dont have the docs published to https://github.com/trinodb/trino-admin yet @mosabua can we get that done?

Reply:
This would be https://trinodb.github.io/trino-admin/

It is from the `gh-pages` branch in this repo, published automatically by GitHub Pages.

Reply:
Sure .. but this is still presto-admin.. we should cut a 2.13 that updates everything to Trino and deploys a new version of the site as well.